### PR TITLE
beautify chat code

### PR DESCRIPTION
**Pull request by GrotheFAF**

### CODE REVIEW

File: src/chat/_avatarWidget.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def __init__(self, users = [], idavatar = 0, parent=None, *args, **kwargs):
         
         self.grid = QtGui.QGridLayout(self)
         self.userlist = None
-        #self.grid.addWidget(self.userlist, 0, 0)
```
Comment (duk3luk3) [inline src/chat/_avatarWidget.py:24]: Is there any point in leaving in these commented-out lines?

**Comment by duk3luk3** [comment]
Looks good to me except for commented out lines, I'd remove those altogether. :+1:

File: src/chat/_avatarWidget.py (hunk 1 above)
Comment (GrotheFAF) [reply src/chat/_avatarWidget.py:24]: These out-commented-lines are not 'my work' so I left them untouched (aside of beautification).

**Comment by duk3luk3** [comment]
Could you please rebase this and and push again so the tests can run again?

**Comment by GrotheFAF** [comment]
I rebased and pushed (with force ;-)

**Comment by duk3luk3** [comment]
Please rebase this and remove commented-out code :-)

**Comment by GrotheFAF** [comment]
I can do that.. but I have no idea who commented the code out and why

**Comment by GrotheFAF** [comment]
so all done...

**Comment by duk3luk3** [comment]
Looks good to me, could you please rebase it on develop one last time? :-)

**Comment by GrotheFAF** [comment]
rebased

**Pull request merged**